comment_generation/app.py

```
@@ -0,0 +1,88 @@
+import json
+from openai import OpenAI
+import boto3
+from botocore.exceptions import ClientError
+
+def get_openai_api_key():
+    secret_name = "prod/DBC/OpenAI"
```

Ruff (0.14.10)

Possible hardcoded password assigned to: "secret_name"

(S105)